src/open_inwoner/configurations/admin.py

```diff
@@ -339,6 +354,76 @@ def _get_column(props):
             ),
         )
 
+    def javascript_preview(self, obj):
+        """Show preview of the uploaded script"""
+        if obj.custom_javascript:
+            try:
+                obj.custom_javascript.seek(0)
+                content = obj.custom_javascript.read().decode("utf-8")
+                obj.custom_javascript.seek(0)
+
+                # Show first 5000 characters
+                preview = content[:5000] + ("..." if len(content) > 5000 else "")
```

Comment:
You can use `truncatechars` with `from django.template.defaultfilters import truncatechars` and then do 
```suggestion
                preview = truncatechars(content, 5000)
```

Reply:
But do we want to keep the preview?

Reply:
Since we are not offering any previews anywhere else in the Admin, I am removing this whole functionality.